config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-test-infra.yaml

```diff
@@ -29,7 +29,7 @@ postsubmits:
 
 periodics:
 - name: metrics-bigquery
-  cron: "03 0-23/3 * * *"  # like interval: 6h, but explicitly starting at 00:03 UTC
```

Comment:
AFAICT this was actually every 3 hours https://elmah.io/tools/cron-parser/#03_0-23/3_*_*_*

Reply:
comment drift :)